charts/mimir-distributed/values.yaml

```
@@ -219,6 +249,11 @@ mimir:
 
     {{- end }}
 
+    {{- if .Values.store_gateway.rollout_operator.enabled -}}
```

Needs new line before new section

```suggestion
    {{- if .Values.store_gateway.rollout_operator.enabled }}
```